main.c

```diff
@@ -4,8 +4,7 @@
 #include "groupTestIncludes.h"
 
 int main(int argc, char** argv) {
-    fileName = malloc(sizeof(char) * 10);
```

Comment:
group 1 will still malloc memory for the original file and you should either request through command line an output file or when we call the script required it. id suggest asking the user which file to output into and make it a new file.